tests/plugin_e2e/test_key_plugins_e2e.py

```
@@ -1094,6 +1115,51 @@ def test_pi_live_package_install_and_extension_smoke(tmp_path: Path):
+                    handle.write(json.dumps(args) + "\\n")
+            if "context" in args:
+                payload = {{
+                    "rendered_markdown": "# Pi smoke Context Bundle\\n\\nInjected by fake nmem.",
+                    "content": "# Pi smoke Context Bundle fallback",
+                }}
+            elif "wm" in args:
+                payload = {{"exists": True, "content": "# Working Memory fallback"}}
+            else:
+                payload = {{}}
+            sys.stdout.write(json.dumps(payload))
+            '''
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)